src/Message/FifoMessage.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace ZfrEbWorker\Message;
+
+/**
+ * @author Michaël Gallego
+ */
+class FifoMessage extends Message implements FifoMessageInterface
```

Comment:
I think we could add $groupId and $deduplicationId as optional params to `Message` class instead of creating a new specific class.